SS_prelim.tpl

```diff
@@ -1387,6 +1390,8 @@
   ParmTrace << endl;
 
   //  SS_Label_Info_6.10 #Preliminary calcs done; Ready for estimation
+  cout << "done" << endl; // evaluating biology calculations once
```

Comment:
Change "done" to "Finished biological calculations"? Not sure on the exact text that would be needed but done is not very descript.